ERCS/erc-____.md

```diff
@@ -0,0 +1,492 @@
+---
+eip: ____
```

Comment:
```suggestion
eip: 7700
```
Assigning next sequential EIP/ERC/RIP number.
Numbering changed to sequential from 7500.

Please also update the filename.

Reply:
Fixed in https://github.com/ethereum/ERCs/pull/404/commits/7af50f32660600c2321aa5ba29af24322bd8c304